pkg/controller/account/account_controller.go

```diff
@@ -71,10 +72,15 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileAccount{
-		Client:           utils.NewClientWithMetricsOrDie(log, mgr, controllerName),
-		scheme:           mgr.GetScheme(),
-		awsClientBuilder: awsclient.GetAWSClient,
+	return &wrappers.TimedReconciler{
```

Comment:
I'm hesitant to wrap the boiler plate code with a dependency in another library. If this is functionality that needs to be default for all the operators maybe it should be contributed upstream in the SDK?

Reply:
> I'm hesitant to wrap the boiler plate code

I'm open to suggestions on a different pattern to use here, but this isn't boilerplate code. We're already instrumenting the controller-runtime client and adding an AWS client to it. This is effectively the same thing as the former.

> maybe it should be contributed upstream in the SDK?

We've talked about that, and it's a good possibility, but it would be quite a long time before we would be able to use that here: It has to get proposed, reviewed, iterated on, merged, and released; and *then* a-a-o has to upgrade to whatever release of operator-sdk it winds up in.